torch/_inductor/codegen/triton.py

```diff
@@ -860,7 +864,7 @@ def indexing(
                 # indirect indexing
                 cse_var = self.cse.varname_map[var.name]
                 mask_vars.update(cse_var.mask_vars)
-            elif var.name.startswith("s"):
+            elif var.name.startswith("s") or var.name.startswith("ps"):
```

Comment:
```suggestion
            elif var.name.startswith(("s", "ps")):
```
More readable and efficient.